### ISSUE TITLE
Catch too many arguments for properties

### ISSUE THREAD

**Issue by PCManticore** [PyCQA/pylint#3006]
For example, this property cannot accept any argument, so we should emit an error about it.
```
class A:
   @property
   def json(self, **kwargs):
       return kwargs

print(A().json)
```
